src/widget/city_overlay_other.c

```diff
@@ -1,6 +1,7 @@
 #include "city_overlay_other.h"
 
 #include "building/model.h"
+#include "core/config.h"
 #include "city/finance.h"
 #include "core/calc.h"
```

Comment:
Alphabetical order :)
```suggestion
#include "city/finance.h"
#include "core/calc.h"
#include "core/config.h"
```

Reply:
It seems I don't know the alphabet anymore 😛